### ISSUE TITLE
when 'target' field is used, the code breaks. Otherwise, code works fine

### ISSUE THREAD

**Issue by deniseOme** [opentargets/platform#537]
From one of our users.
When using Python to get the same results for this endpoint

`api.opentargets.io/v3/platform/public/evidence/filter?size=1000&datasource=chembl&fields=disease.efo_info&fields=drug&fields=evidence&fields=target&fields=access_level&target=ENSG00000005844&view=sec:known_drug&expandefo=true`

This code breaks:

```python
from opentargets import OpenTargetsClient
ot = OpenTargetsClient()
drugs_for_target = ot.get_evidence_for_target('ITGAL',fields=['disease.efo_info','drug','evidence','target','access_level'])
print(drugs_for_target)
```

What the user has got:

0 Results found | parameters: {'target': 'ENSG00000005844', 'fields': ['disease.efo_info', 
---

These code snippets work:

- `drugs_for_target = c.get_evidence_for_target('ITGAL',fields=['disease','drug','evidenc
e'], size=10000)`

Returns:
260 Results found | parameters: {'fields': ['disease', 'drug', 'evidence'], 'target': u'EN
SG00000005844', 'size': 10000}

- `drugs_for_target = c.get_evidence_for_target('ITGAL',fields=['disease.efo_info','drug'
,'evidence'])`.

In summary: the same query adding `'target'` does not work.

**Comment by afaulconbridge** [comment]
No differences to me:
```python
Python 2.7.15+ (default, Nov 27 2018, 23:36:35) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from opentargets import OpenTargetsClient
>>> ot = OpenTargetsClient()
/home/faulcon/Desktop/537/venv/local/lib/python2.7/site-packages/opentargets/conn.py:299: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.api_specs = yaml.load(self.swagger_yaml)
>>> drugs_for_target = ot.get_evidence_for_target('ITGAL',fields=['disease.efo_info','drug','evidence','target','access_level'])
>>> print(drugs_for_target)
0 Results found | parameters: {'fields': ['disease.efo_info', 'drug', 'evidence', 'target', 'access_level'], 'target': u'ENSG00000005844', 'size': 1000}
>>> drugs_for_target = ot.get_evidence_for_target('ITGAL',fields=['disease.efo_info','drug','evidence','access_level'])
>>> print(drugs_for_target)
0 Results found | parameters: {'fields': ['disease.efo_info', 'drug', 'evidence', 'access_level'], 'target': u'ENSG00000005844', 'size': 1000}
>>> 
```
Closing as insufficient information, please add more description if need to reopen.
